packages/dds/tree/src/test/testChange.ts

```diff
@@ -228,7 +228,10 @@ export const TestChange = {
 	toDelta,
 	isEmpty,
 	codec,
-	codecs: makeCodecFamily([[1, codec]]),
+	codecs: makeCodecFamily([
+		[1, codec],
+		[2, codec],
```

Comment:
Why do we need a new one of these?

Reply:
`makeEditManagerCodecs` is used with test codec as a child family.

In general one consequence of not resolving versions top-down is that even when families are structured such that they don't depend on the exact logic encoding/decoding 'subdata', they still take a dependency on the versions supported for encoding that data (`makeEditManagerCodecs` depends on the fact that the codec family for its changes has versions 1 and 2). Pretty awkward IMO.

Reply:
Aggreed